pkg/apis/streaming/v1alpha1/processor_types.go

```diff
@@ -46,6 +47,9 @@ type ProcessorSpec struct {
 	Inputs []StreamBinding `json:"inputs"`
 	// Outputs references an ordered list of streams to bind as outputs
 	Outputs []StreamBinding `json:"outputs"`
+
```

Comment:
Is `// +optional` still a thing or does it have no consequences whatsoever?
(This comment should go on line 44, but I can't put it there.)

Reply:
it's used when generating the openapi schema on the crd. Added to ProcessorSpec, but should probably be reviewed globally.